examples/blender/blender.py

```diff
@@ -38,14 +35,18 @@ async def main(subnet_tag, payment_driver=None, payment_network=None, show_usage
     )
 
     async def worker(ctx: WorkContext, tasks):
+        def new_script():
+            return ctx.new_script(timeout=timedelta(minutes=10))
```

Comment:
Perhaps instead of this function let's just move the default timeout to a "constant" and initialize new scripts directly?

Reply:
what do you mean "directly" ?

Reply:
What I meant was to remove the internal function `new_script`, move the `timedelta` value to a "constant" and use `ctx.new_script` directly in both places where its needed.